### ISSUE TITLE
Add a serial bootloader

### ISSUE THREAD

**Issue by giacomo892** [krzysztofkuczek/QCZEK_LRS#12]
Adding a bootloader with serial support will allow us to update the firmware without using STLINK, but even using serial passthrough of FC

**Comment by krzysztofkuczek** [comment]
good point, but it's only possible if stm8 is equipped with rom boot loader.
In other cases, it would be impossible due to limitation of flash memory...
